pages/common/sftp.md

```diff
@@ -0,0 +1,20 @@
+# sftp
+
+> Secure file transfer program.
+> Copy files between hosts using Secure File Transfer Protocol over SSH.
```

Comment:
You're right, it's not interactive in all cases. The second one is though, but I don't mind implicitly building on the first example instead of being explicit about the whole behavior in every example description.